src/engine.py

```
@@ -4381,6 +4396,39 @@ async def _handle_channel_talking_finished(self, event: dict) -> None:
+                                transcript = await stt.flush_speech(call_id, pipeline.options_summary().get("stt", {}))
+                                if transcript:
+                                    logger.debug("Early STT flush returned transcript", call_id=call_id, transcript_preview=transcript[:50])
+                                    tq = getattr(self, "_pipeline_transcript_queues", {}).get(call_id)
+                                    if tq:
+                                        try:
+                                            tq.put_nowait(transcript)
+                                        except asyncio.QueueFull:
+                                            pass
+                                    else:
+                                        logger.warning("Pipeline transcript queue not found for early flush", call_id=call_id)
+                            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)